docs/plugins/development/git.md

```diff
@@ -60,25 +60,129 @@ This plugin will significantly slow down the speed of data preparation, especial
 
 ### contributors
 
-- Type: `boolean`
+- Type: `boolean | ContributorsOptions`
+
+  ```ts
+  interface ContributorsOptions {
+    /**
+     * Functions to transform contributors, e.g. remove duplicates ones and sort them.
+     * The input is the contributors collected by this plugin, and the output should be the transformed contributors.
+     */
+    transform?: (
+      contributors: GitContributor[],
+    ) => GitContributor[] | Promise<GitContributor[]>
+
+    /**
+     * List of contributors configurations
+     */
+    list?: ContributorConfig[]
+
+    /**
+     * Whether to add avatar in contributor information
+     * @default false
+     */
+    avatar?: boolean
+  }
+
+  interface ContributorConfig {
+    /**
+     * Contributor's username on the git hosting service
+     */
+    username: string
+    /**
+     * The alias of the contributor,
+     * Since contributors may have different usernames saved in their local git configuration
+     * compared to their usernames on the hosting service, In this case, aliases can be used to
+     * map to the actual usernames.
+     */
+    alias?: string[] | string
+    /**
+     * The avatar url of the contributor.
+     *
+     * If the git hosting service is `github`, it can be ignored and left blank,
+     * as the plugin will automatically fill it in.
+     */
+    avatar?: string
+    /**
+     * The url of the contributor
+     *
+     * If the git hosting service is `github`, it can be ignored and left blank,
+     * as the plugin will automatically fill it in.
+     */
+    url?: string
+  }
+  ```
 
 - Default: `true`
 
 - Details:
 
   Whether to collect page contributors or not.
 
-### transformContributors
-
-- Type: `(contributors: GitContributor[]) => GitContributor[]`
+### changelogs
+
+- Type: `false | ChangelogOptions`
+
+  ```ts
+  interface ChangelogOptions {
+    /**
+     * Maximum number of changelogs
+     */
+    maxCount?: number
+
+    /**
+     * The url of the git repository, e.g: https://github.com/vuepress/ecosystem
+     */
+    repoUrl?: string
+
+    /**
+     * Commit url pattern
+     * Default: ':repo/commit/:hash'
+     *
+     * - `:repo` - The url of the git repository
+     * - `:hash` - Hash of the commit record
+     */
+    commitUrlPattern?: string
+
+    /**
+     * Issue url pattern
+     * Default: ':repo/issues/:issue'
+     *
+     * - `:repo` - The url of the git repository
+     * - `:issue` - Id of the issue
+     */
+    issueUrlPattern?: string
+
+    /**
+     * Tag url pattern
+     * Default: ':repo/releases/tag/:tag'
+     *
+     * - `:repo` - The url of the git repository
+     * - `:tag` - Name of the tag
+     */
+    tagUrlPattern?: string
+  }
+  ```
+
+- Default: `false`
 
 - Details:
 
-  A function to transform the contributors.
+  Whether to collect page changelogs or not.
 
-  The input is the contributors collected by this plugin, and the output should be the transformed contributors.
+### enabled
```

Comment:
I prefer to call this `filter` and accept only function values.

The true and false seem like a redundant option to me.